### DISCUSSION TITLE
PVP

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#3255] [Suggestions]
If you are going to keep damage reduction in pvp for the long run which it looks like you are then pvp needs a increase in damage reduction.
20% was added at 49 and 59 before raid gear and now players have raid gear so are negating the 20% reduction because they just hit harder then when it was implemented

**Comment by Annamaria-CC** [comment]
As you have noticed we have made some changes to the PvP buffs/debuffs.
